### DISCUSSION TITLE
The Backrooms

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1447] [Maps and Content Showcase]
This is a simple, small, experimental map that contains 4 different "backroom" stages within itself.
Since the map is using RE's ID system, each time you start a match on it, it will spawn you on one of the 4 levels (Yellow, Building complex, Parking lot, Pool).
But there are no weapons or any items to pick up, so i suggest to play this with the Arena mutator on. 

![20231024145945](https://github.com/redeclipse/base/assets/24359960/22da5cd3-6916-45c9-a309-6925c61a9741)
![20231024150009](https://github.com/redeclipse/base/assets/24359960/61cd18ac-ed13-4f5a-a7a5-2271a963bf53)
![20231024145647](https://github.com/redeclipse/base/assets/24359960/ab56bd1f-13e9-44b5-bc23-5b32ea2d73c2)
![20231024145738](https://github.com/redeclipse/base/assets/24359960/e3d400ed-28a8-457f-a306-fdb71c1d1f71)
![20231024145810](https://github.com/redeclipse/base/assets/24359960/763dccc7-06ca-4aac-852b-d6babb562e5f)
![20231024145915](https://github.com/redeclipse/base/assets/24359960/e3fdb5c5-1468-4ed2-86e8-074f2f8b15af)
![20231024145921](https://github.com/redeclipse/base/assets/24359960/03461d62-52f5-4d39-a7c3-ae7b9ced80c8)

I might update it with more levels in the future, but i'm not sure.
It's not a "Serious" style of map, more like a fun side project.
Hope you all enjoy it!

(i can't seem to be able to upload the zip file for some reason, so, no download on github itself for now, so i'll do it on onedrive, ugh)

https://1drv.ms/u/s!Ak845KFr9Yhv_3ikkXG_jjAhwRjC?e=JH0wTa

**Comment by IceflowRE** [comment]
License?

**Reply by SniperGoth** [reply]
Is it necessary? I never had to put any, besides, this doesn't use any custom assets.

**Reply by IceflowRE** [reply]
If you do not put any license (or allow it) it would be not allowed to host the map.

**Comment by qreeves** [comment]
> If you do not put any license (or allow it) it would be not allowed to host the map.

In RE the agreement is the license is CC-BY-SA unless otherwise specified by the author.

**Reply by IceflowRE** [reply]
This covers only `Content included in the game` , the discussions isn't covered by that license. If it is, it should be explictely noted.

I know that in this community, this is known, but for newcommers or in case of a disput i would not bet on it.
